docs/50-tutorials/troubleshooting.mdx

```diff
@@ -28,6 +28,13 @@ VRCFury.
 * If there are errors related to Gesture Manager, ensure it is up to date in the VCC or get a fresh copy from their github.
 * If there are errors related to some other script, try deleting it if you don't need it.
 
+## Toggles or other properties "stick" or do not work at all
+
+* This is most likely because of a [broken Write Defaults mix](/technical/wd#mixed-write-defaults). 
+* Check any of your VRCFury component's headers for a `W` next to the version number. This indicates that you clicked `Skip and stop asking` when VRCFury detected a (likely) broken WD mix and asked whether or not it should try to fix it for you automatically. 

```

Comment:
The debug line is for internal use only. They should check the avatar root for a Fix Write Defaults component instead.

Reply:
Removed and reworded it.